tests/ui/conftest.py

```diff
@@ -1,20 +1,72 @@
 import datetime
 import json
 import os
+import random
 import urlparse
 
 import jwt
 import pytest
 import requests
 from django.core.management import call_command
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
-from olympia.amo.tests import create_switch
+from olympia import amo
+from olympia.addons.forms import icons
+from olympia.addons.models import AddonUser, Preview
+from olympia.addons.utils import generate_addon_guid
+from olympia.amo.tests import (
+    addon_factory,
+    create_switch,
+    user_factory,
+    version_factory,
+)
+from olympia.constants.applications import APPS, FIREFOX
+from olympia.constants.base import (
+    ADDON_EXTENSION,
+    ADDON_PERSONA,
+    STATUS_PUBLIC
+)
+from olympia.landfill.collection import generate_collection
+from olympia.landfill.generators import generate_themes
+from olympia.reviews.models import Review
+from olympia.users.models import UserProfile
 from pytest_django import live_server_helper
 
 
+@pytest.fixture
+def load_fixtures(request, my_base_url):
+    """
+    This fixture checks the base url. If it determines that a live server is
+    running, it will load the fixtures provided from the text.
+
+    If it determines that a live server is not running, it will load no
+    fixtures.
+
+    :returns: pytest fixture
+    """
+
+    # Checks if localhost is set in basegit url env variable
+    if 'localhost' in my_base_url:
```

Comment:
This is probably okay, but 'localhost' in the URL doesn't necessarily mean we're using a liveserver. I would consider adding an attribute to pytestconfig in the liveserver fixture that we can use to check if we're using it.